x-team/auto-ui
# Change tooltip position

**Release Type:** *Non-Breaking Feature* 


Fixes https://x-team-internal.atlassian.net/browse/XP-2411

## Description
- I decided to change the position of the tooltip, that way we fix this issue and we are doing a safe change. I tried to modify the `Dropdown` component but I realized that the ticket can be a `Breaking change` if we do this. I think this fix can avoid more lateral issues on the website.



## Checklist

**Before submitting a pull request,** please make sure the following is done:



## Related PRs



*N/A*

## Steps to Test or Reproduce



1- Go to Admin Page
2- Click on Tabular view
3- Hover the texts in the column `Avail. Date`
4- You should be able to see the tooltip at the bottom of the text

## Impacted Areas in Application
- **Admin Page - Tabular view** 



## Screenshots

![tooltip](https://user-images.githubusercontent.com/5392202/47944165-524ad580-ded0-11e8-9241-d3d1c9386c46.gif)
